<head>
<title>MetNetComp</title>
</head>
<body>
<table>
<tr>
<td><img src="MetNetComp.png" width="100"> </td>
<td> &nbsp </td>
<td><b><font color="green">MetNetComp Database</font></b><br><br>
at least stoichioemetrically feasible gene deletion strategies for constraint-based metabolic networks<br>
</td>
</tr>
</table>
<br>Model : <a href="http://bigg.ucsd.edu/models/e_coli_core">e_coli_core</a> [1].<br>
Target metabolite : actp_c<br><br>
Gene deletion strategy for growth-coupled production (at least stoichioemetrically feasible)<br>
&nbsp Gene deletion size : 48<br>
<font color="green">&nbsp Gene deletion: 
<a href=https://www.genome.jp/entry/eco:b1241>b1241</a> <a href=https://www.genome.jp/entry/eco:b0351>b0351</a> <a href=https://www.genome.jp/entry/eco:b1849>b1849</a> <a href=https://www.genome.jp/entry/eco:b3115>b3115</a> <a href=https://www.genome.jp/entry/eco:b2296>b2296</a> <a href=https://www.genome.jp/entry/eco:b0474>b0474</a> <a href=https://www.genome.jp/entry/eco:b0726>b0726</a> <a href=https://www.genome.jp/entry/eco:b2587>b2587</a> <a href=https://www.genome.jp/entry/eco:b0356>b0356</a> <a href=https://www.genome.jp/entry/eco:b1478>b1478</a> <a href=https://www.genome.jp/entry/eco:b3603>b3603</a> <a href=https://www.genome.jp/entry/eco:b2975>b2975</a> <a href=https://www.genome.jp/entry/eco:b3925>b3925</a> <a href=https://www.genome.jp/entry/eco:b4232>b4232</a> <a href=https://www.genome.jp/entry/eco:b2492>b2492</a> <a href=https://www.genome.jp/entry/eco:b0904>b0904</a> <a href=https://www.genome.jp/entry/eco:b4154>b4154</a> <a href=https://www.genome.jp/entry/eco:b1818>b1818</a> <a href=https://www.genome.jp/entry/eco:b1611>b1611</a> <a href=https://www.genome.jp/entry/eco:b4122>b4122</a> <a href=https://www.genome.jp/entry/eco:b1612>b1612</a> <a href=https://www.genome.jp/entry/eco:b3528>b3528</a> <a href=https://www.genome.jp/entry/eco:b1852>b1852</a> <a href=https://www.genome.jp/entry/eco:b0810>b0810</a> <a href=https://www.genome.jp/entry/eco:b1761>b1761</a> <a href=https://www.genome.jp/entry/eco:b1524>b1524</a> <a href=https://www.genome.jp/entry/eco:b0485>b0485</a> <a href=https://www.genome.jp/entry/eco:b1812>b1812</a> <a href=https://www.genome.jp/entry/eco:b4077>b4077</a> <a href=https://www.genome.jp/entry/eco:b2029>b2029</a> <a href=https://www.genome.jp/entry/eco:b4015>b4015</a> <a href=https://www.genome.jp/entry/eco:b1380>b1380</a> <a href=https://www.genome.jp/entry/eco:b2133>b2133</a> <a href=https://www.genome.jp/entry/eco:b4014>b4014</a> <a href=https://www.genome.jp/entry/eco:b2976>b2976</a> <a href=https://www.genome.jp/entry/eco:b1479>b1479</a> <a href=https://www.genome.jp/entry/eco:b1603>b1603</a> <a href=https://www.genome.jp/entry/eco:b3962>b3962</a> <a href=https://www.genome.jp/entry/eco:b3114>b3114</a> <a href=https://www.genome.jp/entry/eco:b3952>b3952</a> <a href=https://www.genome.jp/entry/eco:b0903>b0903</a> <a href=https://www.genome.jp/entry/eco:b0767>b0767</a> <a href=https://www.genome.jp/entry/eco:b3403>b3403</a> <a href=https://www.genome.jp/entry/eco:b1702>b1702</a> <a href=https://www.genome.jp/entry/eco:b1676>b1676</a> <a href=https://www.genome.jp/entry/eco:b1854>b1854</a> <a href=https://www.genome.jp/entry/eco:b0724>b0724</a> <a href=https://www.genome.jp/entry/eco:b0729>b0729</a> 
</font><br>
&nbsp Computed by: TrimGdel [2]<br><br>
When growth rate is maximized,<br>
<font color="green">
&nbsp Growth Rate : 0.405103 (mmol/gDw/h)<br>
&nbsp Minimum Production Rate : 13.843047 (mmol/gDw/h)<br><br>
</font>
Substrate: (mmol/gDw/h)<br>
&nbsp EX_o2_e : 12.031934<br>
&nbsp EX_pi_e : 10.184726<br>
&nbsp EX_glc__D_e : 10.000000<br>
&nbsp EX_nh4_e : 2.208944<br>
<br>
Product: (mmol/gDw/h)<br>
&nbsp EX_h2o_e : 26.719964<br>
&nbsp EX_h_e : 13.274931<br>
&nbsp EX_co2_e : 9.925987<br>
&nbsp Transport : 8.694475<br>
<br>Visualization<br>
&nbsp 1. <a href="../json/e_coli_core&actp_c.json">Download JSON file</a>.<br>
&nbsp 2. Go to <a href="https://escher.github.io/#/app?map=e_coli_core.Core%20metabolism&tool=Builder&model=e_coli_core"> Escher site</a> [3].<br>
&nbsp 3. Select "Data > Load reaction data" and apply the downloaded file.<br><br>
References<br>
[1] Orth, J. D., Fleming, R. M., Palsson, B. Ø. (2010). 
Reconstruction and use of microbial metabolic networks: the core Escherichia coli metabolic model as an educational guide. EcoSal plus, 4(1).<br>
[2] Tamura, T. (2022). Trimming gene deletion strategies for growth-coupled production in constraint-based metabolic networks: TrimGdel. IEEE/ACM Transactions on Computational Biology and Bioinformatics.<br>
[3] King, Z. A., Dräger, A., Ebrahim, A., Sonnenschein, N., Lewis, N. E., & Palsson, B. O. (2015). Escher: a web application for building, sharing, and embedding data-rich visualizations of biological pathways. PLoS computational biology, 11(8), e1004321.<br>
</body>
