<head>
<title>MetNetComp</title>
</head>
<body>
<table>
<tr>
<td><a href="https://metnetcomp.github.io/database_beta/indexFiles/index.html"><img src="MetNetComp.png" width="100"></a> </td>
<td> &nbsp </td>
<td><b><font color="green">MetNetComp Database / Minimal gene deletions</font></b><br><br>
Minimal gene deletions for simulation-based growth-coupled production. You can also see <a href=iHN637&dtmp_c&core.html>maximal gene deletions</a>.<br><br>
</td>
</tr>
</table>
<br>Model : <a href="http://bigg.ucsd.edu/models/iHN637">iHN637</a> [1].<br>
Target metabolite : <a href="http://bigg.ucsd.edu/models/iHN637/metabolites/dtmp_c">dtmp_c</a><br>
List of minimal gene deletion strategies <a href="../csv/iHN637&dtmp_c.csv" download="iHN637&dtmp_c.csv">(Download)</a><br><br>
Gene deletion strategy (4 of 4: <a href="iHN637&dtmp_c&1.html">See next</a>) for growth-coupled production (at least stoichioemetrically feasible)<br>
&nbsp Gene deletion size : 10<br>
<font color="green">&nbsp Gene deletion: 
CLJU_RS19990 CLJU_RS06265 CLJU_RS03525 CLJU_RS19980 CLJU_RS15035 CLJU_RS12480 CLJU_RS18510 CLJU_RS11995 CLJU_RS01595 CLJU_RS18530 &nbsp <a href="../alternativeGenes/alternative_genes&iHN637&dtmp_c&4&minimal.html">(List of alternative genes)</a>
</font><br>
&nbsp Computed by: <a href="https://github.com/MetNetComp/Gdel-minRN">gDel-minRN</a> [2] and randTrimGdel [3]<br><br>
When growth rate is maximized,<br>
<font color="green">
&nbsp Growth Rate : 0.099169 (mmol/gDw/h)<br>
&nbsp Minimum Production Rate : 0.007250 (mmol/gDw/h)<br><br>
</font>
Substrate: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_fru_e">EX_fru_e</a> : 5.000000<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_nh4_e">EX_nh4_e</a> : 0.847416<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_pi_e">EX_pi_e</a> : 0.122152<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_so4_e">EX_so4_e</a> : 0.099137<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_k_e">EX_k_e</a> : 0.018959<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_fe2_e">EX_fe2_e</a> : 0.001371<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_mg2_e">EX_mg2_e</a> : 0.000843<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_ca2_e">EX_ca2_e</a> : 0.000506<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_cl_e">EX_cl_e</a> : 0.000506<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_fol_e">EX_fol_e</a> : 0.000088<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_cu2_e">EX_cu2_e</a> : 0.000069<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_mn2_e">EX_mn2_e</a> : 0.000067<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_ribflv_e">EX_ribflv_e</a> : 0.000044<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_zn2_e">EX_zn2_e</a> : 0.000033<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_ni2_e">EX_ni2_e</a> : 0.000031<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_cobalt2_e">EX_cobalt2_e</a> : 0.000025<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_thm_e">EX_thm_e</a> : 0.000022<br>
<br>
Product: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_co2_e">EX_co2_e</a> : 8.885898<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_etoh_e">EX_etoh_e</a> : 7.981993<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h2o_e">EX_h2o_e</a> : 3.057787<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h_e">EX_h_e</a> : 0.663735<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_ac_e">EX_ac_e</a> : 0.098716<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_met__L_e">EX_met__L_e</a> : 0.080411<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/DM_succ_c">DM_succ_c</a> : 0.053783<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/Auxiliary production reaction">Auxiliary production reaction</a> : 0.007250<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/DM_mththf_c">DM_mththf_c</a> : 0.000221<br>
<br>Visualization<br>
&nbsp 1. <a href="../json/iHN637&dtmp_c&minimal&4.json" download="../json/iHN637&dtmp_c&minimal&4.json">Download JSON file</a>.<br>
&nbsp 2. Go to <a href="https://escher.github.io/#/"> Escher site</a> [4].<br><br>
References<br>
[1] Norsigian, C. J., Pusarla, N., McConn, J. L., Yurkovich, J. T., Dräger, A., Palsson, B. O., & King, Z. (2020). BiGG Models 2020: multi-strain genome-scale models and expansion across the phylogenetic tree. Nucleic acids research, 48(D1), D402-D406.<br>
[2] Tamura, T., Muto-Fujita, A., Tohsato, Y., & Kosaka, T. Gene Deletion Algorithms for Minimum Reaction Network Design by Mixed-integer Linear Programming for Metabolite Production in Constraint-based Models: gDel_minRN. Journal of Computational Biology, in press.<br>
[3] Tamura, T. MetNetComp: database for minimal and maximal gene deletion strategies for growth-coupled production for genome-scale metabolic networks, to be submitted.<br>
[4] King, Z. A., Dräger, A., Ebrahim, A., Sonnenschein, N., Lewis, N. E., & Palsson, B. O. (2015). Escher: a web application for building, sharing, and embedding data-rich visualizations of biological pathways. PLoS computational biology, 11(8), e1004321.<br>
<br><hr>Last updated: 02-Feb-2023<br>
<a href="../indexFiles/contact.html">Contact</a><br>
</body>
