<head>
<title>MetNetComp</title>
</head>
<body>
<table>
<tr>
<td><a href="https://metnetcomp.github.io/database1/indexFiles/index.html"><img src="MetNetComp.png" width="100"></a> </td>
<td> &nbsp </td>
<td><b><font color="green">MetNetComp Database / Minimal gene deletions</font></b><br><br>
Minimal gene deletions for simulation-based growth-coupled production. You can also see <a href=iHN637&acald_c&core.html>maximal gene deletions</a>.<br><br>
</td>
</tr>
</table>
<br>Model : <a href="http://bigg.ucsd.edu/models/iHN637">iHN637</a> [1].<br>
Target metabolite : <a href="http://bigg.ucsd.edu/models/iHN637/metabolites/acald_c">acald_c</a><br>
List of minimal gene deletion strategies <a href="../csv/iHN637&acald_c.csv" download="iHN637&acald_c.csv">(Download)</a><br><br>
Gene deletion strategy (21 of 59: <a href="iHN637&acald_c&22.html">See next</a>) for growth-coupled production (at least stoichioemetrically feasible)<br>
&nbsp Gene deletion size : 9<br>
<font color="green">&nbsp Gene deletion: 
CLJU_RS19990 CLJU_RS05830 CLJU_RS08100 CLJU_RS08095 CLJU_RS03525 CLJU_RS11990 CLJU_RS18495 CLJU_RS06260 CLJU_RS18530 &nbsp <a href="../alternativeGenes/alternative_genes&iHN637&acald_c&21&minimal.html">(List of alternative genes)</a>
</font><br>
&nbsp Computed by: RandTrimGdel [1] (<a href="https://github.com/MetNetComp/Gdel-minRN">Step 1</a>, <a href="https://github.com/MetNetComp/RandTrimGdel-Step2">Step 2</a>)<br><br>
When growth rate is maximized,<br>
<font color="green">
&nbsp Growth Rate : 0.143531 (mmol/gDw/h)<br>
&nbsp Minimum Production Rate : 0.041973 (mmol/gDw/h)<br><br>
</font>
Substrate: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_fru_e">EX_fru_e</a> : 5.000000<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_nh4_e">EX_nh4_e</a> : 1.216004<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_pi_e">EX_pi_e</a> : 0.166302<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_so4_e">EX_so4_e</a> : 0.153978<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_k_e">EX_k_e</a> : 0.027440<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_fe2_e">EX_fe2_e</a> : 0.001984<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_mg2_e">EX_mg2_e</a> : 0.001219<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_ca2_e">EX_ca2_e</a> : 0.000732<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_cl_e">EX_cl_e</a> : 0.000732<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_fol_e">EX_fol_e</a> : 0.000128<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_cu2_e">EX_cu2_e</a> : 0.000100<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_mn2_e">EX_mn2_e</a> : 0.000097<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_ribflv_e">EX_ribflv_e</a> : 0.000064<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_zn2_e">EX_zn2_e</a> : 0.000048<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_ni2_e">EX_ni2_e</a> : 0.000045<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_cobalt2_e">EX_cobalt2_e</a> : 0.000036<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_thm_e">EX_thm_e</a> : 0.000032<br>
<br>
Product: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_co2_e">EX_co2_e</a> : 8.405007<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_etoh_e">EX_etoh_e</a> : 7.044286<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h2o_e">EX_h2o_e</a> : 4.425649<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h_e">EX_h_e</a> : 0.939663<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_ac_e">EX_ac_e</a> : 0.153369<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_met__L_e">EX_met__L_e</a> : 0.126875<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/DM_succ_c">DM_succ_c</a> : 0.077843<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/Auxiliary production reaction">Auxiliary production reaction</a> : 0.041973<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/DM_mththf_c">DM_mththf_c</a> : 0.000320<br>
<br>Visualization<br>
&nbsp 1. <a href="../json/iHN637&acald_c&minimal&21.json" download="../json/iHN637&acald_c&minimal&21.json">Download JSON file</a>.<br>
&nbsp 2. Go to <a href="https://escher.github.io/#/"> Escher site</a> [3].<br><br>
References<br>
[1] Norsigian, C. J., Pusarla, N., McConn, J. L., Yurkovich, J. T., Dräger, A., Palsson, B. O., & King, Z. (2020). BiGG Models 2020: multi-strain genome-scale models and expansion across the phylogenetic tree. Nucleic acids research, 48(D1), D402-D406.<br>
[2] Tamura, T., Muto-Fujita, A., Tohsato, Y., & Kosaka, T. Gene Deletion Algorithms for Minimum Reaction Network Design by Mixed-integer Linear Programming for Metabolite Production in Constraint-based Models: gDel_minRN. Journal of Computational Biology, in press.<br>
[3] King, Z. A., Dräger, A., Ebrahim, A., Sonnenschein, N., Lewis, N. E., & Palsson, B. O. (2015). Escher: a web application for building, sharing, and embedding data-rich visualizations of biological pathways. PLoS computational biology, 11(8), e1004321.<br>
<br><hr>Last updated: 05-Apr-2023<br>
<a href="../indexFiles/contact.html">Contact</a><br>
</body>
