<head>
<title>MetNetComp</title>
</head>
<body>
<table>
<tr>
<td><a href="https://metnetcomp.github.io/database_beta/indexFiles/index.html"><img src="MetNetComp.png" width="100"></a> </td>
<td> &nbsp </td>
<td><b><font color="green">MetNetComp Database / Minimal gene deletions</font></b><br><br>
Minimal gene deletions for simulation-based growth-coupled production. You can also see <a href=iLJ478&23dhmb_c&core.html>maximal gene deletions</a>.<br><br>
</td>
</tr>
</table>
<br>Model : <a href="http://bigg.ucsd.edu/models/iLJ478">iLJ478</a> [1].<br>
Target metabolite : <a href="http://bigg.ucsd.edu/models/iLJ478/metabolites/23dhmb_c">23dhmb_c</a><br>
List of minimal gene deletion strategies <a href="../csv/iLJ478&23dhmb_c.csv" download="iLJ478&23dhmb_c.csv">(Download)</a><br><br>
Gene deletion strategy (1 of 3: <a href="iLJ478&23dhmb_c&2.html">See next</a>) for growth-coupled production (at least stoichioemetrically feasible)<br>
&nbsp Gene deletion size : 3<br>
<font color="green">&nbsp Gene deletion: 
<a href=https://www.genome.jp/entry/tma:TM0273>TM0273</a> <a href=https://www.genome.jp/entry/tma:TM1640>TM1640</a> <a href=https://www.genome.jp/entry/tma:TM1217>TM1217</a> &nbsp <a href="../alternativeGenes/alternative_genes&iLJ478&23dhmb_c&1&minimal.html">(List of alternative genes)</a>
</font><br>
&nbsp Computed by: <a href="https://github.com/MetNetComp/Gdel-minRN">gDel-minRN</a> [2] and randTrimGdel [3]<br><br>
When growth rate is maximized,<br>
<font color="green">
&nbsp Growth Rate : 0.088681 (mmol/gDw/h)<br>
&nbsp Minimum Production Rate : 2.483374 (mmol/gDw/h)<br><br>
</font>
Substrate: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h2o_e">EX_h2o_e</a> : 6.829497<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_s_e">EX_s_e</a> : 4.000000<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_malt_e">EX_malt_e</a> : 2.794000<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_nh4_e">EX_nh4_e</a> : 0.919722<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_pi_e">EX_pi_e</a> : 0.068175<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/SK_apoACP_c">SK_apoACP_c</a> : 0.000248<br>
<br>
Product: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h2_e">EX_h2_e</a> : 10.057473<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_co2_e">EX_co2_e</a> : 8.222214<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h_e">EX_h_e</a> : 8.186233<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_ac_e">EX_ac_e</a> : 4.868113<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h2s_e">EX_h2s_e</a> : 3.978463<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/Auxiliary production reaction">Auxiliary production reaction</a> : 2.483374<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/SK_gcald_c">SK_gcald_c</a> : 0.004434<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/SK_5mta_c">SK_5mta_c</a> : 0.000621<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/SK_pap_c">SK_pap_c</a> : 0.000248<br>
<br>Visualization<br>
&nbsp 1. <a href="../json/iLJ478&23dhmb_c&minimal&1.json" download="../json/iLJ478&23dhmb_c&minimal&1.json">Download JSON file</a>.<br>
&nbsp 2. Go to <a href="https://escher.github.io/#/"> Escher site</a> [4].<br><br>
References<br>
[1] Norsigian, C. J., Pusarla, N., McConn, J. L., Yurkovich, J. T., Dräger, A., Palsson, B. O., & King, Z. (2020). BiGG Models 2020: multi-strain genome-scale models and expansion across the phylogenetic tree. Nucleic acids research, 48(D1), D402-D406.<br>
[2] Tamura, T., Muto-Fujita, A., Tohsato, Y., & Kosaka, T. Gene Deletion Algorithms for Minimum Reaction Network Design by Mixed-integer Linear Programming for Metabolite Production in Constraint-based Models: gDel_minRN. Journal of Computational Biology, in press.<br>
[3] Tamura, T. MetNetComp: database for minimal and maximal gene deletion strategies for growth-coupled production for genome-scale metabolic networks, to be submitted.<br>
[4] King, Z. A., Dräger, A., Ebrahim, A., Sonnenschein, N., Lewis, N. E., & Palsson, B. O. (2015). Escher: a web application for building, sharing, and embedding data-rich visualizations of biological pathways. PLoS computational biology, 11(8), e1004321.<br>
<br><hr>Last updated: 05-Jan-2023<br>
<a href="../indexFiles/contact.html">Contact</a><br>
</body>
