<head>
<title>MetNetComp</title>
</head>
<body>
<table>
<tr>
<td><img src="MetNetComp.png" width="100"> </td>
<td> &nbsp </td>
<td><b><font color="green">MetNetComp Database</font></b><br><br>
at least stoichioemetrically feasible gene deletion strategies for constraint-based metabolic networks<br>
</td>
</tr>
</table>
<br>Model : <a href="http://bigg.ucsd.edu/models/iML1515">iML1515</a> [1].<br>
Target metabolite : glyc_e<br><br>
Gene deletion strategy for growth-coupled production (at least stoichioemetrically feasible)<br>
&nbsp Gene deletion size : 28<br>
<font color="green">&nbsp Gene deletion: 
b3553 b1241 b0351 b3926 b0871 b2925 b2097 b1004 b3713 b1109 b0046 b3236 b3946 b0825 b1033 b1603 b4381 b0755 b3612 b2492 b0904 b1380 b2660 b0514 b1771 b0606 b2278 b4209 
</font><br>
&nbsp Computed by: TrimGdel [2]<br><br>
When growth rate is maximized,<br>
<font color="green">
&nbsp Growth Rate : 0.489242 (mmol/gDw/h)<br>
&nbsp Minimum Production Rate : 1.240179 (mmol/gDw/h)<br><br>
</font>
Substrate: (mmol/gDw/h)<br>
&nbsp EX_o2_e : 34.608524<br>
&nbsp EX_glc__D_e : 10.000000<br>
&nbsp EX_nh4_e : 5.283775<br>
&nbsp EX_pi_e : 0.471926<br>
&nbsp EX_fe3_e : 0.129960<br>
&nbsp EX_so4_e : 0.123201<br>
&nbsp EX_k_e : 0.095497<br>
&nbsp EX_mg2_e : 0.004244<br>
&nbsp EX_ca2_e : 0.002547<br>
&nbsp EX_cl_e : 0.002547<br>
&nbsp EX_cu2_e : 0.000347<br>
&nbsp EX_mn2_e : 0.000338<br>
&nbsp EX_zn2_e : 0.000167<br>
&nbsp EX_ni2_e : 0.000158<br>
&nbsp EX_cobalt2_e : 0.000012<br>
<br>
Product: (mmol/gDw/h)<br>
&nbsp EX_h2o_e : 47.934796<br>
&nbsp EX_co2_e : 36.289893<br>
&nbsp EX_h_e : 4.625313<br>
&nbsp EX_glyc_e : 1.209653<br>
&nbsp EX_fe2_e : 0.122102<br>
&nbsp DM_5drib_c : 0.000110<br>
&nbsp DM_4crsol_c : 0.000109<br>
&nbsp EX_meoh_e : 0.000001<br>
&nbsp DM_amob_c : 0.000001<br>
&nbsp DM_mththf_c : 0.000000<br>
&nbsp EX_3hpp_e : -0.000000<br>
&nbsp EX_quin_e : 0.000000<br>
&nbsp EX_metglcur_e : 0.000000<br>
&nbsp EX_rfamp_e : -0.000000<br>
&nbsp EX_4abzglu_e : 0.000000<br>
<br>Visualization<br>
&nbsp - <a href="../json/iML1515&glyc_e.json">Download JSON file</a>.<br>
&nbsp - The json file can be applied to <a href="https://escher.github.io/#/"> Escher</a> [3]. (Currently, the map of iML1515 seems not available.)<br>
References<br>
[1] Norsigian, C. J., Pusarla, N., McConn, J. L., Yurkovich, J. T., Dräger, A., Palsson, B. O., & King, Z. (2020).
BiGG Models 2020: multi-strain genome-scale models and expansion across the phylogenetic tree. Nucleic acids research, 48(D1), D402-D406.<br>
[2] Tamura, T. (2022). Trimming gene deletion strategies for growth-coupled production in constraint-based metabolic networks: TrimGdel. IEEE/ACM Transactions on Computational Biology and Bioinformatics.<br>
[3] King, Z. A., Dräger, A., Ebrahim, A., Sonnenschein, N., Lewis, N. E., & Palsson, B. O. (2015). Escher: a web application for building, sharing, and embedding data-rich visualizations of biological pathways. PLoS computational biology, 11(8), e1004321.<br>
</body>
