<head>
<title>MetNetComp</title>
</head>
<body>
<table>
<tr>
<td><a href="https://metnetcomp.github.io/database_beta/indexFiles/index.html"><img src="MetNetComp.png" width="100"></a> </td>
<td> &nbsp </td>
<td><b><font color="green">MetNetComp Database / Minimal gene deletions</font></b><br><br>
Minimal gene deletions for simulation-based growth-coupled production. You can also see <a href=iML1515&2mahmp_c&core.html>maximal gene deletions</a>.<br><br>
</td>
</tr>
</table>
<br>Model : <a href="http://bigg.ucsd.edu/models/iML1515">iML1515</a> [1].<br>
Target metabolite : 2mahmp_c<br><br>
Gene deletion strategy for growth-coupled production (at least stoichioemetrically feasible)<br>
&nbsp Gene deletion size : 4<br>
<font color="green">&nbsp Gene deletion: 
<a href=https://www.genome.jp/entry/eco:b3114>b3114</a> <a href=https://www.genome.jp/entry/eco:b0903>b0903</a> <a href=https://www.genome.jp/entry/eco:b3951>b3951</a> <a href=https://www.genome.jp/entry/eco:b0529>b0529</a> 
</font><br>
&nbsp Computed by: TrimGdel [2]<br><br>
When growth rate is maximized,<br>
<font color="green">
&nbsp Growth Rate : 0.764866 (mmol/gDw/h)<br>
&nbsp Minimum Production Rate : 0.755931 (mmol/gDw/h)<br><br>
</font>
Substrate: (mmol/gDw/h)<br>
&nbsp EX_o2_e : 22.815923<br>
&nbsp EX_nh4_e : 10.528276<br>
&nbsp EX_glc__D_e : 10.000000<br>
&nbsp EX_pi_e : 2.249656<br>
&nbsp EX_so4_e : 0.192608<br>
&nbsp EX_k_e : 0.149296<br>
&nbsp EX_fe2_e : 0.012285<br>
&nbsp EX_mg2_e : 0.006635<br>
&nbsp EX_cl_e : 0.003981<br>
&nbsp EX_ca2_e : 0.003981<br>
&nbsp EX_cu2_e : 0.000542<br>
&nbsp EX_mn2_e : 0.000529<br>
&nbsp EX_zn2_e : 0.000261<br>
&nbsp EX_ni2_e : 0.000247<br>
&nbsp EX_cobalt2_e : 0.000019<br>
<br>
Product: (mmol/gDw/h)<br>
&nbsp EX_h2o_e : 50.315630<br>
&nbsp EX_co2_e : 24.070196<br>
&nbsp EX_h_e : 8.539752<br>
&nbsp Auxiliary production reaction : 0.755931<br>
&nbsp DM_5drib_c : 0.000172<br>
&nbsp DM_4crsol_c : 0.000171<br>
<br>Visualization<br>
&nbsp 1. <a href="../json/iML1515&2mahmp_c.json">Download JSON file</a> (converted to iJO1366).<br>
&nbsp 2. Go to <a href="https://escher.github.io/#/app?map=iJO1366.Central metabolism&tool=Builder&model=iJO1366"> Escher map</a> [3].<br>
&nbsp 3. Select "Data > Load reaction data" and apply the downloaded file.<br><br>
References<br>
[1] Orth, J. D., Fleming, R. M., Palsson, B. Ø. (2010). 
Reconstruction and use of microbial metabolic networks: the core Escherichia coli metabolic model as an educational guide. EcoSal plus, 4(1).<br>
[2] Tamura, T. (2022). Trimming gene deletion strategies for growth-coupled production in constraint-based metabolic networks: TrimGdel. IEEE/ACM Transactions on Computational Biology and Bioinformatics.<br>
[3] King, Z. A., Dräger, A., Ebrahim, A., Sonnenschein, N., Lewis, N. E., & Palsson, B. O. (2015). Escher: a web application for building, sharing, and embedding data-rich visualizations of biological pathways. PLoS computational biology, 11(8), e1004321.<br>
</body>
