<head>
<title>MetNetComp</title>
</head>
<body>
<table>
<tr>
<td><a href="https://metnetcomp.github.io/database_beta/indexFiles/index.html"><img src="MetNetComp.png" width="100"></a> </td>
<td> &nbsp </td>
<td><b><font color="green">MetNetComp Database</font></b><br><br>
at least stoichioemetrically feasible gene deletion strategies for constraint-based metabolic networks<br>
</td>
</tr>
</table>
<br>Model : <a href="http://bigg.ucsd.edu/models/iML1515">iML1515</a> [1].<br>
Target metabolite : h2mb4p_c<br><br>
Gene deletion strategy for growth-coupled production (at least stoichioemetrically feasible)<br>
&nbsp Gene deletion size : 24<br>
<font color="green">&nbsp Gene deletion: 
<a href=https://www.genome.jp/entry/eco:b1241>b1241</a> <a href=https://www.genome.jp/entry/eco:b0351>b0351</a> <a href=https://www.genome.jp/entry/eco:b4069>b4069</a> <a href=https://www.genome.jp/entry/eco:b3115>b3115</a> <a href=https://www.genome.jp/entry/eco:b1849>b1849</a> <a href=https://www.genome.jp/entry/eco:b2296>b2296</a> <a href=https://www.genome.jp/entry/eco:b1612>b1612</a> <a href=https://www.genome.jp/entry/eco:b1611>b1611</a> <a href=https://www.genome.jp/entry/eco:b4122>b4122</a> <a href=https://www.genome.jp/entry/eco:b3616>b3616</a> <a href=https://www.genome.jp/entry/eco:b3589>b3589</a> <a href=https://www.genome.jp/entry/eco:b4381>b4381</a> <a href=https://www.genome.jp/entry/eco:b0511>b0511</a> <a href=https://www.genome.jp/entry/eco:b0115>b0115</a> <a href=https://www.genome.jp/entry/eco:b0755>b0755</a> <a href=https://www.genome.jp/entry/eco:b3612>b3612</a> <a href=https://www.genome.jp/entry/eco:b0529>b0529</a> <a href=https://www.genome.jp/entry/eco:b2492>b2492</a> <a href=https://www.genome.jp/entry/eco:b0904>b0904</a> <a href=https://www.genome.jp/entry/eco:b0515>b0515</a> <a href=https://www.genome.jp/entry/eco:b1813>b1813</a> <a href=https://www.genome.jp/entry/eco:b1517>b1517</a> <a href=https://www.genome.jp/entry/eco:b1206>b1206</a> <a href=https://www.genome.jp/entry/eco:b2287>b2287</a> 
</font><br>
&nbsp Computed by: TrimGdel [2]<br><br>
When growth rate is maximized,<br>
<font color="green">
&nbsp Growth Rate : 0.410143 (mmol/gDw/h)<br>
&nbsp Minimum Production Rate : 1.573157 (mmol/gDw/h)<br><br>
</font>
Substrate: (mmol/gDw/h)<br>
&nbsp EX_fe2_e : 1000.000000<br>
&nbsp EX_h_e : 996.343237<br>
&nbsp EX_o2_e : 279.580327<br>
&nbsp EX_glc__D_e : 10.000000<br>
&nbsp EX_nh4_e : 4.429506<br>
&nbsp EX_pi_e : 3.541940<br>
&nbsp EX_so4_e : 0.103282<br>
&nbsp EX_k_e : 0.080057<br>
&nbsp EX_mg2_e : 0.003558<br>
&nbsp EX_ca2_e : 0.002135<br>
&nbsp EX_cl_e : 0.002135<br>
&nbsp EX_cu2_e : 0.000291<br>
&nbsp EX_mn2_e : 0.000283<br>
&nbsp EX_zn2_e : 0.000140<br>
&nbsp EX_ni2_e : 0.000132<br>
&nbsp EX_cobalt2_e : 0.000010<br>
<br>
Product: (mmol/gDw/h)<br>
&nbsp EX_fe3_e : 999.993413<br>
&nbsp EX_h2o_e : 547.218410<br>
&nbsp EX_co2_e : 32.389269<br>
&nbsp Transport : 1.573157<br>
&nbsp EX_ac_e : 0.599390<br>
&nbsp EX_fum_e : 0.427694<br>
&nbsp DM_mththf_c : 0.000184<br>
&nbsp DM_5drib_c : 0.000092<br>
&nbsp DM_4crsol_c : 0.000091<br>
&nbsp EX_meoh_e : 0.000001<br>
&nbsp DM_amob_c : 0.000001<br>
&nbsp EX_3hpp_e : -0.000000<br>
&nbsp EX_quin_e : 0.000000<br>
&nbsp EX_metglcur_e : 0.000000<br>
&nbsp EX_rfamp_e : -0.000000<br>
&nbsp EX_4abzglu_e : 0.000000<br>
<br>Visualization<br>
&nbsp - <a href="../json/iML1515&h2mb4p_c.json">Download JSON file</a>.<br>
&nbsp - The json file can be applied to <a href="https://escher.github.io/#/"> Escher</a> [3]. (Currently, the map of iML1515 seems not available.)<br>
References<br>
[1] Norsigian, C. J., Pusarla, N., McConn, J. L., Yurkovich, J. T., Dräger, A., Palsson, B. O., & King, Z. (2020).
BiGG Models 2020: multi-strain genome-scale models and expansion across the phylogenetic tree. Nucleic acids research, 48(D1), D402-D406.<br>
[2] Tamura, T. (2022). Trimming gene deletion strategies for growth-coupled production in constraint-based metabolic networks: TrimGdel. IEEE/ACM Transactions on Computational Biology and Bioinformatics.<br>
[3] King, Z. A., Dräger, A., Ebrahim, A., Sonnenschein, N., Lewis, N. E., & Palsson, B. O. (2015). Escher: a web application for building, sharing, and embedding data-rich visualizations of biological pathways. PLoS computational biology, 11(8), e1004321.<br>
</body>
