<head>
<title>MetNetComp</title>
</head>
<body>
<table>
<tr>
<td><a href="https://metnetcomp.github.io/database_beta/indexFiles/index.html"><img src="MetNetComp.png" width="100"></a> </td>
<td> &nbsp </td>
<td><b><font color="green">MetNetComp Database / Minimal gene deletions</font></b><br><br>
Minimal gene deletions for simulation-based growth-coupled production. You can also see <a href=iML1515&ump_c&core.html>maximal gene deletions</a>.<br><br>
</td>
</tr>
</table>
<br>Model : <a href="http://bigg.ucsd.edu/models/iML1515">iML1515</a> [1].<br>
Target metabolite : ump_c<br><br>
Gene deletion strategy for growth-coupled production (at least stoichioemetrically feasible)<br>
&nbsp Gene deletion size : 33<br>
<font color="green">&nbsp Gene deletion: 
<a href=https://www.genome.jp/entry/eco:b1241>b1241</a> <a href=https://www.genome.jp/entry/eco:b0351>b0351</a> <a href=https://www.genome.jp/entry/eco:b4069>b4069</a> <a href=https://www.genome.jp/entry/eco:b2744>b2744</a> <a href=https://www.genome.jp/entry/eco:b3614>b3614</a> <a href=https://www.genome.jp/entry/eco:b0910>b0910</a> <a href=https://www.genome.jp/entry/eco:b4151>b4151</a> <a href=https://www.genome.jp/entry/eco:b3115>b3115</a> <a href=https://www.genome.jp/entry/eco:b1849>b1849</a> <a href=https://www.genome.jp/entry/eco:b2296>b2296</a> <a href=https://www.genome.jp/entry/eco:b2781>b2781</a> <a href=https://www.genome.jp/entry/eco:b1612>b1612</a> <a href=https://www.genome.jp/entry/eco:b1611>b1611</a> <a href=https://www.genome.jp/entry/eco:b4122>b4122</a> <a href=https://www.genome.jp/entry/eco:b1759>b1759</a> <a href=https://www.genome.jp/entry/eco:b3616>b3616</a> <a href=https://www.genome.jp/entry/eco:b3589>b3589</a> <a href=https://www.genome.jp/entry/eco:b4374>b4374</a> <a href=https://www.genome.jp/entry/eco:b0675>b0675</a> <a href=https://www.genome.jp/entry/eco:b4138>b4138</a> <a href=https://www.genome.jp/entry/eco:b4123>b4123</a> <a href=https://www.genome.jp/entry/eco:b0621>b0621</a> <a href=https://www.genome.jp/entry/eco:b4381>b4381</a> <a href=https://www.genome.jp/entry/eco:b2406>b2406</a> <a href=https://www.genome.jp/entry/eco:b0115>b0115</a> <a href=https://www.genome.jp/entry/eco:b0529>b0529</a> <a href=https://www.genome.jp/entry/eco:b2492>b2492</a> <a href=https://www.genome.jp/entry/eco:b0904>b0904</a> <a href=https://www.genome.jp/entry/eco:b3918>b3918</a> <a href=https://www.genome.jp/entry/eco:b1912>b1912</a> <a href=https://www.genome.jp/entry/eco:b1517>b1517</a> <a href=https://www.genome.jp/entry/eco:b1206>b1206</a> <a href=https://www.genome.jp/entry/eco:b2278>b2278</a> 
</font><br>
&nbsp Computed by: TrimGdel [2]<br><br>
When growth rate is maximized,<br>
<font color="green">
&nbsp Growth Rate : 0.387526 (mmol/gDw/h)<br>
&nbsp Minimum Production Rate : 0.947873 (mmol/gDw/h)<br><br>
</font>
Substrate: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_fe2_e">EX_fe2_e</a> : 1000.000000<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h_e">EX_h_e</a> : 989.376575<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_o2_e">EX_o2_e</a> : 279.008829<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_glc__D_e">EX_glc__D_e</a> : 10.000000<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_nh4_e">EX_nh4_e</a> : 6.964481<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_pi_e">EX_pi_e</a> : 1.321683<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_so4_e">EX_so4_e</a> : 0.097587<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_k_e">EX_k_e</a> : 0.075642<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_mg2_e">EX_mg2_e</a> : 0.003362<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_ca2_e">EX_ca2_e</a> : 0.002017<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_cl_e">EX_cl_e</a> : 0.002017<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_cu2_e">EX_cu2_e</a> : 0.000275<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_mn2_e">EX_mn2_e</a> : 0.000268<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_zn2_e">EX_zn2_e</a> : 0.000132<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_ni2_e">EX_ni2_e</a> : 0.000125<br>
<br>
Product: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_fe3_e">EX_fe3_e</a> : 999.993776<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h2o_e">EX_h2o_e</a> : 546.506876<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_co2_e">EX_co2_e</a> : 26.656960<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_ac_e">EX_ac_e</a> : 1.239397<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_succ_e">EX_succ_e</a> : 1.147312<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/Auxiliary production reaction">Auxiliary production reaction</a> : 0.947873<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/DM_oxam_c">DM_oxam_c</a> : 0.743203<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_thym_e">EX_thym_e</a> : 0.070143<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/DM_mththf_c">DM_mththf_c</a> : 0.000174<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/DM_5drib_c">DM_5drib_c</a> : 0.000087<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/DM_4crsol_c">DM_4crsol_c</a> : 0.000086<br>
<br>Visualization<br>
&nbsp 1. <a href="../json/iML1515&ump_c.json">Download JSON file</a> (converted to iJO1366).<br>
&nbsp 2. Go to <a href="https://escher.github.io/#/app?map=iJO1366.Central metabolism&tool=Builder&model=iJO1366"> Escher map</a> [3].<br>
&nbsp 3. Select "Data > Load reaction data" and apply the downloaded file.<br><br>
References<br>
[1] Orth, J. D., Fleming, R. M., Palsson, B. Ø. (2010). 
Reconstruction and use of microbial metabolic networks: the core Escherichia coli metabolic model as an educational guide. EcoSal plus, 4(1).<br>
[2] Tamura, T. (2022). Trimming gene deletion strategies for growth-coupled production in constraint-based metabolic networks: TrimGdel. IEEE/ACM Transactions on Computational Biology and Bioinformatics.<br>
[3] King, Z. A., Dräger, A., Ebrahim, A., Sonnenschein, N., Lewis, N. E., & Palsson, B. O. (2015). Escher: a web application for building, sharing, and embedding data-rich visualizations of biological pathways. PLoS computational biology, 11(8), e1004321.<br>
</body>
