<head>
<title>MetNetComp</title>
</head>
<body>
<table>
<tr>
<td><a href="https://metnetcomp.github.io/database_beta/indexFiles/index.html"><img src="MetNetComp.png" width="100"></a> </td>
<td> &nbsp </td>
<td><b><font color="green">MetNetComp Database</font></b><br><br>
at least stoichioemetrically feasible gene deletion strategies for constraint-based metabolic networks<br>
</td>
</tr>
</table>
<br>Model : <a href="http://bigg.ucsd.edu/models/iMM904">iMM904</a> [1].<br>
Target metabolite : leu__L_m<br><br>
Gene deletion strategy for growth-coupled production (at least stoichioemetrically feasible)<br>
&nbsp Gene deletion size : 34<br>
<font color="green">&nbsp Gene deletion: 
<a href=https://www.genome.jp/entry/sce:YBR068C>YBR068C</a> <a href=https://www.genome.jp/entry/sce:YDR046C>YDR046C</a> <a href=https://www.genome.jp/entry/sce:YBR069C>YBR069C</a> <a href=https://www.genome.jp/entry/sce:YGR087C>YGR087C</a> <a href=https://www.genome.jp/entry/sce:YLR044C>YLR044C</a> <a href=https://www.genome.jp/entry/sce:YLR134W>YLR134W</a> <a href=https://www.genome.jp/entry/sce:YDR380W>YDR380W</a> <a href=https://www.genome.jp/entry/sce:YDL080C>YDL080C</a> <a href=https://www.genome.jp/entry/sce:YDR508C>YDR508C</a> <a href=https://www.genome.jp/entry/sce:YLR027C>YLR027C</a> <a href=https://www.genome.jp/entry/sce:YJR057W>YJR057W</a> <a href=https://www.genome.jp/entry/sce:YCL025C>YCL025C</a> <a href=https://www.genome.jp/entry/sce:YKR039W>YKR039W</a> <a href=https://www.genome.jp/entry/sce:YPR062W>YPR062W</a> <a href=https://www.genome.jp/entry/sce:YIL053W>YIL053W</a> <a href=https://www.genome.jp/entry/sce:YER062C>YER062C</a> <a href=https://www.genome.jp/entry/sce:YGR191W>YGR191W</a> <a href=https://www.genome.jp/entry/sce:YDR300C>YDR300C</a> <a href=https://www.genome.jp/entry/sce:YMR250W>YMR250W</a> <a href=https://www.genome.jp/entry/sce:YER065C>YER065C</a> <a href=https://www.genome.jp/entry/sce:YML059C>YML059C</a> <a href=https://www.genome.jp/entry/sce:YJL121C>YJL121C</a> <a href=https://www.genome.jp/entry/sce:YLR348C>YLR348C</a> <a href=https://www.genome.jp/entry/sce:YGR055W>YGR055W</a> <a href=https://www.genome.jp/entry/sce:YHL036W>YHL036W</a> <a href=https://www.genome.jp/entry/sce:YKL141W>YKL141W</a> <a href=https://www.genome.jp/entry/sce:YDR178W>YDR178W</a> <a href=https://www.genome.jp/entry/sce:YLR017W>YLR017W</a> <a href=https://www.genome.jp/entry/sce:YDL085W>YDL085W</a> <a href=https://www.genome.jp/entry/sce:YMR145C>YMR145C</a> <a href=https://www.genome.jp/entry/sce:YEL046C>YEL046C</a> <a href=https://www.genome.jp/entry/sce:YDR402C>YDR402C</a> <a href=https://www.genome.jp/entry/sce:YOR155C>YOR155C</a> <a href=https://www.genome.jp/entry/sce:YGL224C>YGL224C</a> 
</font><br>
&nbsp Computed by: TrimGdel [2]<br><br>
When growth rate is maximized,<br>
<font color="green">
&nbsp Growth Rate : 0.103156 (mmol/gDw/h)<br>
&nbsp Minimum Production Rate : 1.748229 (mmol/gDw/h)<br><br>
</font>
Substrate: (mmol/gDw/h)<br>
&nbsp EX_glc__D_e : 10.000000<br>
&nbsp EX_nh4_e : 2.347039<br>
&nbsp EX_o2_e : 2.000000<br>
&nbsp EX_pi_e : 0.020394<br>
&nbsp EX_so4_e : 0.007974<br>
<br>
Product: (mmol/gDw/h)<br>
&nbsp EX_h_e : 25.059359<br>
&nbsp EX_h2o_e : 12.544246<br>
&nbsp EX_succ_e : 9.438549<br>
&nbsp EX_ac_e : 3.737185<br>
&nbsp Transport : 1.748229<br>
<br>Visualization<br>
&nbsp 1. <a href="../json/iMM904&leu__L_m.json">Download JSON file</a>.<br>
&nbsp 2. Go to <a href="https://escher.github.io/#/app?map=iMM904.Central carbon metabolism&tool=Builder&model=iMM904">Escher site</a> [3].<br>
&nbsp 3. Select "Data > Load reaction data" and apply the downloaded file.<br><br>
References<br>
[1] Norsigian, C. J., Pusarla, N., McConn, J. L., Yurkovich, J. T., Dräger, A., Palsson, B. O., & King, Z. (2020).
BiGG Models 2020: multi-strain genome-scale models and expansion across the phylogenetic tree. Nucleic acids research, 48(D1), D402-D406.<br>
[2] Tamura, T. (2022). Trimming gene deletion strategies for growth-coupled production in constraint-based metabolic networks: TrimGdel. IEEE/ACM Transactions on Computational Biology and Bioinformatics.<br>
[3] King, Z. A., Dräger, A., Ebrahim, A., Sonnenschein, N., Lewis, N. E., & Palsson, B. O. (2015). Escher: a web application for building, sharing, and embedding data-rich visualizations of biological pathways. PLoS computational biology, 11(8), e1004321.<br>
</body>
