<head>
<title>MetNetComp</title>
</head>
<body>
<table>
<tr>
<td><a href="https://metnetcomp.github.io/database_beta/indexFiles/index.html"><img src="MetNetComp.png" width="100"></a> </td>
<td> &nbsp </td>
<td><b><font color="green">MetNetComp Database / Minimal gene deletions</font></b><br><br>
Minimal gene deletions for simulation-based growth-coupled production. You can also see <a href=iMM904&2mbtoh_e&core.html>maximal gene deletions</a>.<br><br>
</td>
</tr>
</table>
<br>Model : <a href="http://bigg.ucsd.edu/models/iMM904">iMM904</a> [1].<br>
Target metabolite : <a href="http://bigg.ucsd.edu/models/iMM904/metabolites/2mbtoh_e">2mbtoh_e</a><br>
List of minimal gene deletion strategies <a href="../csv/iMM904&2mbtoh_e.csv" download="iMM904&2mbtoh_e.csv">(Download)</a><br><br>
Gene deletion strategy (3 of 3: <a href="iMM904&2mbtoh_e&1.html">See next</a>) for growth-coupled production (at least stoichioemetrically feasible)<br>
&nbsp Gene deletion size : 10<br>
<font color="green">&nbsp Gene deletion: 
<a href=https://www.genome.jp/entry/sce:YHR104W>YHR104W</a> <a href=https://www.genome.jp/entry/sce:YMR083W>YMR083W</a> <a href=https://www.genome.jp/entry/sce:YKL192C>YKL192C</a> <a href=https://www.genome.jp/entry/sce:YDR272W>YDR272W</a> <a href=https://www.genome.jp/entry/sce:YIL155C>YIL155C</a> <a href=https://www.genome.jp/entry/sce:YOL151W>YOL151W</a> <a href=https://www.genome.jp/entry/sce:YLR348C>YLR348C</a> <a href=https://www.genome.jp/entry/sce:YDL085W>YDL085W</a> <a href=https://www.genome.jp/entry/sce:YMR145C>YMR145C</a> <a href=https://www.genome.jp/entry/sce:YML120C>YML120C</a> &nbsp <a href="../alternativeGenes/alternative_genes&iMM904&2mbtoh_e&3&minimal.html">(List of alternative genes)</a>
</font><br>
&nbsp Computed by: <a href="https://github.com/MetNetComp/Gdel-minRN">gDel-minRN</a> [2] and randTrimGdel [3]<br><br>
When growth rate is maximized,<br>
<font color="green">
&nbsp Growth Rate : 0.202236 (mmol/gDw/h)<br>
&nbsp Minimum Production Rate : 0.064291 (mmol/gDw/h)<br><br>
</font>
Substrate: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_glc__D_e">EX_glc__D_e</a> : 10.000000<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_nh4_e">EX_nh4_e</a> : 1.131806<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_pi_e">EX_pi_e</a> : 0.039982<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_o2_e">EX_o2_e</a> : 0.022598<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_so4_e">EX_so4_e</a> : 0.015633<br>
<br>
Product: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_co2_e">EX_co2_e</a> : 17.688788<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_etoh_e">EX_etoh_e</a> : 17.301268<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h2o_e">EX_h2o_e</a> : 2.712642<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h_e">EX_h_e</a> : 1.017302<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_2mbtoh_e">EX_2mbtoh_e</a> : 0.087980<br>
<br>Visualization<br>
&nbsp 1. <a href="../json/iMM904&2mbtoh_e&minimal&3.json" download="../json/iMM904&2mbtoh_e&minimal&3.json">Download JSON file</a>.<br>
&nbsp 2. Go to <a href="https://escher.github.io/#/app?map=iMM904.Central carbon metabolism&tool=Builder&model=iMM904"> Escher site</a> [4].<br>
&nbsp 3. Select "Data > Load reaction data" and apply the downloaded file.<br><br>
References<br>
[1] Norsigian, C. J., Pusarla, N., McConn, J. L., Yurkovich, J. T., Dräger, A., Palsson, B. O., & King, Z. (2020). BiGG Models 2020: multi-strain genome-scale models and expansion across the phylogenetic tree. Nucleic acids research, 48(D1), D402-D406.<br>
[2] Tamura, T., Muto-Fujita, A., Tohsato, Y., & Kosaka, T. (2021). Gene Deletion Algorithms for Minimum Reaction Network Design by Mixed-integer Linear Programming for Metabolite Production in Constraint-based Models: gDel_minRN. preprint.<br>
[3] Tamura, T. MetNetComp: database for minimal and maximal gene deletion strategies for growth-coupled production for genome-scale metabolic networks, in preparation.<br>
[4] King, Z. A., Dräger, A., Ebrahim, A., Sonnenschein, N., Lewis, N. E., & Palsson, B. O. (2015). Escher: a web application for building, sharing, and embedding data-rich visualizations of biological pathways. PLoS computational biology, 11(8), e1004321.<br>
<br><hr>Last updated: 28-Nov-2022<br>
<a href="../indexFiles/contact.html">Contact</a><br>
</body>
