<head>
<title>MetNetComp</title>
</head>
<body>
<table>
<tr>
<td><a href="https://metnetcomp.github.io/database_beta/indexFiles/index.html"><img src="MetNetComp.png" width="100"></a> </td>
<td> &nbsp </td>
<td><b><font color="green">MetNetComp Database</font></b><br><br>
Minimal gene deletions for simulation-based growth-coupled production. You can also see <a href=iMM904&alac__S_m&core.html>maximal gene deletions</a>.<br><br>
</td>
</tr>
</table>
<br>Model : <a href="http://bigg.ucsd.edu/models/iMM904">iMM904</a> [1].<br>
Target metabolite : alac__S_m<br><br>
Gene deletion strategy for growth-coupled production (at least stoichioemetrically feasible)<br>
&nbsp Gene deletion size : 4<br>
<font color="green">&nbsp Gene deletion: 
<a href=https://www.genome.jp/entry/sce:YGR087C>YGR087C</a> <a href=https://www.genome.jp/entry/sce:YLR044C>YLR044C</a> <a href=https://www.genome.jp/entry/sce:YLR134W>YLR134W</a> <a href=https://www.genome.jp/entry/sce:YJL121C>YJL121C</a> 
</font><br>
&nbsp Computed by: TrimGdel [2]<br><br>
When growth rate is maximized,<br>
<font color="green">
&nbsp Growth Rate : 0.164295 (mmol/gDw/h)<br>
&nbsp Minimum Production Rate : 4.376783 (mmol/gDw/h)<br><br>
</font>
Substrate: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_glc__D_e">EX_glc__D_e</a> : 10.000000<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_o2_e">EX_o2_e</a> : 2.000000<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_nh4_e">EX_nh4_e</a> : 0.919469<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_pi_e">EX_pi_e</a> : 0.032481<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_so4_e">EX_so4_e</a> : 0.012700<br>
<br>
Product: (mmol/gDw/h)<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h_e">EX_h_e</a> : 19.782016<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_h2o_e">EX_h2o_e</a> : 13.015281<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_succ_e">EX_succ_e</a> : 7.288968<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/Auxiliary production reaction">Auxiliary production reaction</a> : 4.376783<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_iamoh_e">EX_iamoh_e</a> : 0.610854<br>
&nbsp <a href="http://bigg.ucsd.edu/universal/reactions/EX_for_e">EX_for_e</a> : 0.000849<br>
<br>Visualization<br>
&nbsp 1. <a href="../json/iMM904&alac__S_m&minimal.json">Download JSON file</a>.<br>
&nbsp 2. Go to <a href="https://escher.github.io/#/app?map=iMM904.Central carbon metabolism&tool=Builder&model=iMM904">Escher site</a> [3].<br>
&nbsp 3. Select "Data > Load reaction data" and apply the downloaded file.<br><br>
References<br>
[1] Norsigian, C. J., Pusarla, N., McConn, J. L., Yurkovich, J. T., Dräger, A., Palsson, B. O., & King, Z. (2020).
BiGG Models 2020: multi-strain genome-scale models and expansion across the phylogenetic tree. Nucleic acids research, 48(D1), D402-D406.<br>
[2] Tamura, T. (2022). Trimming gene deletion strategies for growth-coupled production in constraint-based metabolic networks: TrimGdel. IEEE/ACM Transactions on Computational Biology and Bioinformatics.<br>
[3] King, Z. A., Dräger, A., Ebrahim, A., Sonnenschein, N., Lewis, N. E., & Palsson, B. O. (2015). Escher: a web application for building, sharing, and embedding data-rich visualizations of biological pathways. PLoS computational biology, 11(8), e1004321.<br>
</body>
