<head>
<title>MetNetComp</title>
</head>
<body>
<table>
<tr>
<td><a href="https://metnetcomp.github.io/database_beta/indexFiles/index.html"><img src="MetNetComp.png" width="100"></a> </td>
<td> &nbsp </td>
<td><b><font color="green">MetNetComp Database</font></b><br><br>
at least stoichioemetrically feasible gene deletion strategies for constraint-based metabolic networks<br>
</td>
</tr>
</table>
<br>Model : <a href="http://bigg.ucsd.edu/models/iYO844">iYO844</a> [1].<br>
Target metabolite : succ_c<br><br>
Gene deletion strategy for growth-coupled production (at least stoichioemetrically feasible)<br>
&nbsp Gene deletion size : 5<br>
<font color="green">&nbsp Gene deletion: 
<a href=https://www.genome.jp/entry/bsu:BSU04470>BSU04470</a> <a href=https://www.genome.jp/entry/bsu:BSU07570>BSU07570</a> <a href=https://www.genome.jp/entry/bsu:BSU28450>BSU28450</a> <a href=https://www.genome.jp/entry/bsu:BSU16100>BSU16100</a> <a href=https://www.genome.jp/entry/bsu:BSU21770>BSU21770</a> 
</font><br>
&nbsp Computed by: TrimGdel [2]<br><br>
When growth rate is maximized,<br>
<font color="green">
&nbsp Growth Rate : 0.102504 (mmol/gDw/h)<br>
&nbsp Minimum Production Rate : 0.050533 (mmol/gDw/h)<br><br>
</font>
Substrate: (mmol/gDw/h)<br>
&nbsp EX_o2_e : 6.118104<br>
&nbsp EX_glc__D_e : 1.700000<br>
&nbsp EX_nh4_e : 0.821101<br>
&nbsp EX_pi_e : 0.157321<br>
&nbsp EX_k_e : 0.072400<br>
&nbsp EX_so4_e : 0.017458<br>
&nbsp EX_mg2_e : 0.010426<br>
&nbsp EX_fe3_e : 0.000354<br>
&nbsp EX_ca2_e : 0.000329<br>
<br>
Product: (mmol/gDw/h)<br>
&nbsp EX_h2o_e : 8.023653<br>
&nbsp EX_co2_e : 6.331598<br>
&nbsp EX_h_e : 0.768107<br>
&nbsp Transport : 0.050533<br>
&nbsp EX_f6p_e : 0.000000<br>
&nbsp EX_3pg_e : 0.000000<br>
&nbsp EX_3gmp_e : 0.000000<br>
&nbsp EX_3cmp_e : 0.000000<br>
&nbsp EX_3amp_e : 0.000000<br>
<br>Visualization<br>
&nbsp - <a href="../json/iYO844&succ_c.json">Download JSON file</a>.<br>
&nbsp - The json file can be applied to <a href="https://escher.github.io/#/"> Escher</a> [3]. (Currently, the map of iYO844 seems not available.)<br>
References<br>
[1] Norsigian, C. J., Pusarla, N., McConn, J. L., Yurkovich, J. T., Dräger, A., Palsson, B. O., & King, Z. (2020).
BiGG Models 2020: multi-strain genome-scale models and expansion across the phylogenetic tree. Nucleic acids research, 48(D1), D402-D406.<br>
[2] Tamura, T. (2022). Trimming gene deletion strategies for growth-coupled production in constraint-based metabolic networks: TrimGdel. IEEE/ACM Transactions on Computational Biology and Bioinformatics.<br>
[3] King, Z. A., Dräger, A., Ebrahim, A., Sonnenschein, N., Lewis, N. E., & Palsson, B. O. (2015). Escher: a web application for building, sharing, and embedding data-rich visualizations of biological pathways. PLoS computational biology, 11(8), e1004321.<br>
</body>
